src/main/java/core/basesyntax/model/Message.java

```diff
@@ -1,8 +1,26 @@
 package core.basesyntax.model;
 
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "message")
 public class Message {
+    @Id
+    @GeneratedValue (strategy = GenerationType.IDENTITY)
+    @Column (name = "message_id")
     private Long id;
     private String content;
+
+    @OneToMany (fetch = FetchType.EAGER, cascade = CascadeType.ALL)
```

Comment:
do we need eager-fetching here?

Reply:
Don't know - it does work without it, so I removed it...